### PR TITLE
Update the appendix

### PR DESCRIPTION
**Pull request by aofarrel**
* Very in-depth disk space example
* Explanation of miniwdl and Cromwell's configuration files, how to use them, and links to the templates that explain everything within then
* Brief mention of hog factors, which are too complicated to explain in detail but worth linking out to Cromwell's explanation

Broken URL error appears to be a red herring; it's components that have since been updated on main.

### CODE REVIEW

File: 09-appendix-backends.Rmd [hunk 1]
```diff
@@ -59,13 +67,33 @@ Other notes:
...
+* Some HPCs do not support the use of Docker due to security concerns
+
+  * HPCs that do not allow the use of Docker may be able to run WDLs using alternative container technologies such as podman or rootless Docker
+
+* Some HPCs will use `disks` to determine which disk to run on, which can be useful for managing disk space
+
+#### Fred Hutch HPC
+* The Fred Hutch HPC supports the use of Docker, so the `docker` runtime attribute works as expected
+
+* The `memory` and `gpu` runtime attributes are ignored, but `cpu` works as expected
```
Comment (caalo) [inline 09-appendix-backends.Rmd:82]: Let's make sure we change "memory" in the table above to "n/a"!

File: 09-appendix-backends.Rmd [hunk 2]
```diff
@@ -98,21 +126,84 @@ Common pitfalls:
...
+        File? ref_alt         # optional file
+        Array[File] fastqs    # required array of files
+        File tarball_bams     # uncompressed tarball of bam files
+        File compressed_vcfs  # compressed (tar.gz) tarball of VCFs
+        Int? addl_disk_gb     # user-defined integer 
+    }
 
-#### Fred Hutch HPC
-* The Fred Hutch HPC supports the use of Docker, so the `docker` runtime attribute works as expected
+    # ref_genome always exists, so we can size() on it safely. size() returns a float,
```
Comment (caalo) [inline 09-appendix-backends.Rmd:144]: Great example.

**Review by caalo — APPROVED** [review]

File: 09-appendix-backends.Rmd (hunk 1 above)
Comment (caalo) [reply 09-appendix-backends.Rmd:82]: Made the change in this PR https://github.com/fhdsl/WDL_Workflows_Guide/pull/46

**Comment by aofarrel** [comment]
Note to self: Merge this after https://github.com/fhdsl/WDL_Workflows_Guide/pull/46 gets merged!

**Pull request merged**